mce-hbtimer.h

```diff
@@ -0,0 +1,62 @@
+/**
+ * @file mce-hbtimer.h
+ *
+ * Mode Control Entity - Suspend proof timer functionality
+ *
+ * <p>
+ *
+ * Copyright (C) 2014-2015 Jolla Ltd.
+ *
+ * <p>
+ *
+ * @author Simo Piiroinen <simo.piiroinen@jollamobile.com>
+ *
+ * mce is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License
+ * version 2.1 as published by the Free Software Foundation.
+ *
+ * mce is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with mce.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef MCE_HBTIMER_H_
+#define MCE_HBTIMER_H_
+
+#include <stdbool.h>
+#include <glib.h>
+
+#ifdef __cplusplus
+extern "C" {
+#elif 0
+} /* fool JED indentation ... */
```

Comment:
(..insert comment about "don't workaround editor limitations in source code" here..)